packages/utils/src/common/to-camel-case.ts

```diff
@@ -0,0 +1,11 @@
+import { upperCaseFirst } from './upper-case-first'
+
+export function toCamelCase(str: string): string {
+  return str.toLowerCase().split("-").map((n, index) => {
```

Comment:
todo: https://github.com/medusajs/medusa/pull/4378/files#diff-a223870729267da64ffadae7a8461956cf716276aafffdb57314e538a8134894
here we want kebabToKamelCase, it is important to specify it as it does not transform everything to camel case. Also you can reuse the utils I ve shared here ^

Reply:
would it be better to be more generic and convert anything into camelCase? `file name.js`, `file_name.js`, etc

Reply:
I've renamed it to kebabToKamelCase. Agree that it should be generic, but to prevent this PR from being about utils, we can add the generic case when the usecase for it shows up.

Reply:
It should be part of a separate pr I agree, @carlos-r-l-rodrigues do you think of something like 
```ts
.replace(/[^a-zA-Z0-9]+(.)/g, (m, chr) => chr.toUpperCase())
```

Reply:
@riqwan do you think changing the util to `toKebabCase` and replace the implementation by what I ve posted here ^ could be included here? It shouldn't affter the tests except adding `this-is-A-test` => `thisIsATest` for example 😂